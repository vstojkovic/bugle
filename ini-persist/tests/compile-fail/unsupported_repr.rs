use ini_persist::Property;

#[derive(Property)]
#[ini(repr)]
#[repr(C)]
enum Foo {
    Arglebargle,
    GlopGlyf,
}

fn main() {
}
