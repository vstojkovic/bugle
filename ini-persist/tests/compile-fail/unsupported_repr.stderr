error: unsupported repr
 --> tests/compile-fail/unsupported_repr.rs:5:8
  |
5 | #[repr(C)]
  |        ^
